packages/hint-http-cache/tests/tests.ts

```diff
@@ -1,9 +1,9 @@
-import { generateHTMLPage } from '@hint/utils-create-server';
-import { getHintPath, HintTest, testHint } from '@hint/utils-tests-helpers';
+import { generateHTMLPage, getHintPath, testHint } from '@hint/utils-tests-helpers';
```

Comment:
You need to undo this change. `generateHTMLPage` is in `@hint/utils-create-server` now.

Reply:
done